### ISSUE TITLE
Add info about available opensuse package on corp

### ISSUE THREAD

**Issue by RDKRACZ** [PolyMC/polymc.github.io#43]
### Role

I use opensuse Leap 15.3 on my second computer

### Suggestion

Change this:
![Screenshot from 2022-03-13 13-08-44](https://user-images.githubusercontent.com/43277609/158059004-3c0beb97-3581-42f5-8fd7-ecd32353feba.png)

to this:
![Screenshot from 2022-03-13 13-09-24](https://user-images.githubusercontent.com/43277609/158059010-d675ba26-72fe-47af-88e9-19326f35dd19.png)

### Benefit

opensuse users will know about the package
<details>

![Screenshot from 2022-03-13 13-09-45](https://user-images.githubusercontent.com/43277609/158058992-1f0cb143-377c-4a79-add1-758de95792c1.png)

</details>

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Scrumplex** [comment]
We already have a COPR here: https://polymc.org/wiki/installing/linux/fedora/

**Comment by RDKRACZ** [comment]
Yes, that's the one I'm talking about, you transferred the issue when I was in the middle of editing this so github didn't accept the change, imma do it again, wait.

**Comment by RDKRACZ** [comment]
TLDR edit one line of text to also say opensuse since it's available there.

**Comment by ZekeSmith** [comment]
This should be fixed now

**Comment by txtsd** [comment]
Fixed in c384318a1dc22de6eefd416b67f51e4853c3452b
